appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -47,6 +48,7 @@ public fun ComposeContent(json: String, imageLoader: ImageLoader) {
         CompositionLocalProvider(
             LocalImageLoader provides imageLoader,
             LocalLogcues provides Logcues(LoggingLevel.DEBUG),
+            LocalAppcuesConfig provides AppcuesConfig("test-account", "test-app"),
```

Comment:
The updates in this file ensure a non-null AppcuesConfig is available in the compositions used in snapshot testing. There really is no use case right now for passing these values in, so I just used a default config with hardcoded test account/app (which is not used in the UI layer). If we ever needed to pass in custom config from the snapshot tests, we could update these functions to take the config as a param from tests.